backend/solo_rog_api/authentication.py

```diff
@@ -10,16 +10,14 @@
 
 
 class CACAuthenticationBackend(BaseBackend):
-    DODID_RE = re.compile(r"CN=(?:\w+\.){2,3}(?P<dodid>\d{10})")
+    CAC_DN_RE = re.compile(
+        r"CN=(?P<last_name>\w+)\.(?P<first_name>\w+)\.(\w+\.)?(?P<username>\d{10})"
```

Comment:
Can we get the regex to capture the first half of a hyphenated last name? Right now I'm only getting the part of the last name after the hyphen.

Reply:
added hyphen to first and last name regex